src/content/self-hosted/manage/troubleshooting.mdx

```diff
@@ -5,6 +5,45 @@ sidebar_label: Troubleshooting
 id: troubleshooting
 ---
 
+## How to extract logs from Okteto when asking for help
+
+When asking for assistance from Okteto Support, please use the [`okteto logs` command](reference/okteto-cli.mdx#logs) to fetch the logs from your Development Environment.
+
+## Understanding the state of your app: why endpoints aren't present in the CLI or UI
+
+Endpoint links are not present within Okteto if there are no services actively running. If your endpoints are missing, consider the following states and their implications:
```

Comment:
@ifbyol is it running the condition? or running and passing healthchecks?

Reply:
For context, I think checking for `ready` here means healthchecks are passing:
https://github.com/okteto/app/blob/master/backend/pkg/app/component.go#L440

Reply:
For the endpoints, the condition is just `running` state of the component, but the `running` state is not set until all the containers within the pod are ready (readiness probe passed).

If all the containers are not ready, `Booting` is the status and endpoints are not shown in that case

Reply:
Added a "Booting" section. Could you review this?